src/ServiceControl.Infrastructure.Metrics/MeterValues.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ServiceControl.Infrastructure.Metrics
+{
+    public readonly struct MeterValues
+    {
+        public MeterValues(string name, float current, float average15, float average60, float average300)
```

Comment:
The [general guidance for structs](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct) is no more than 16 bytes. This one is 24 bytes. I think it might still be ok though, but we would need to check whether it is copied or not.